pages/index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <main class="max-w-lg mx-auto px-6">
     <add-task @newTask="getTasks" />
-    <transition>
+    <transition name="slide-fade">
```

Comment:
I have now applied transition to individual todos but it works only when todo is deleted. When todo is added this transition is not applied because getTask function is called for getting the entire list of todos.